tools/cluster_mgr.py

```diff
@@ -153,11 +149,180 @@ def main():
         update_id(n)
     print()
 
-    config = build_config(masters)
-    print(f'Pushing config...')
-    push_config(nodes, config)
+    config = build_config_from_list(masters)
+    print(f'Pushing config:\n{config}\n')
+    push_config(config)
     print()
```

Comment:
Maybe before the script didn't grow too big you can try sending the requests in parallel with asyncio? Or is it just for internal use?

Reply:
The same applies to error handling, or is it not a priority as well?

Reply:
It's only for internal checks & tests, so crashing following an error or using async io is not something we should care about :)

Reply:
Hm, okay, then most of my comments don't matter